src/components/card/card.stories.mdx

```diff
@@ -257,18 +198,19 @@ When the card is a focal point and more contrast is desired, you may also attach
 ## Template Properties
 
 - `class` (string): Append a class to the root element.
-- `tag_name` (string, default `'article'`): The root element tag.
-- `header_tag_name` (string, default `'header'`): The header element tag.
-- `footer_tag_name` (string, default `'footer'`): The footer element tag.
+- `heading_id` (string): Add this `id` to the heading element (if present), as well as the `aria-labelledby` property of the card element (unless `aria_labelledby` is already specified).
```

Comment:
I had to read this a couple of times for it to register in my brain. 😅 

Maybe modifying it a bit?

```suggestion
- `heading_id` (string): Adds this `id` to the heading element (if a heading element is present); this also adds the `aria-labelledby` property of the card element (unless `aria_labelledby` is already specified).
```